src/main/java/org/alfresco/repo/search/impl/solr/SolrSQLHttpClient.java

```diff
@@ -200,6 +200,15 @@ public ResultSet executeQuery(SearchParameters searchParameters, String language
                 locales.put(I18NUtil.getLocale());
             }
             body.put("locales", locales);
+
+            // FIXME
```

Comment:
Looks fine, do we have a TAS Rest API test to support this

Reply:
I think filter query / filter query parameter is not supported for /sql api yet, hence no tas rest api test is available for now.

Reply:
Yes, this is a note that we can merge once we have a test to support the change and a green build.